pkg/config/env/mongo.go

```diff
@@ -67,3 +74,52 @@ func MongoConnectionTimeoutWithDefault(value string) string {
 func MongoSSLWithDefault(value string) string {
 	return envy.Get("MONGO_SSL", value)
 }
+
+// MongoConnDetails fetches appropriate environment variables from other
+// Mongo* functions and returns a mongo.ConnDetails according to their
+// values. Otherwise, returns nil and a non-nil error
+func MongoConnDetails() (*mongo.ConnDetails, error) {
+	host, err := MongoHost()
+	if err != nil {
+		return nil, err
+	}
+	portStr, err := MongoPort()
+	if err != nil {
+		return nil, err
+	}
+	port, err := strconv.Atoi(portStr)
+	if err != nil {
+		return nil, err
+	}
+
+	user, err := MongoUser()
+	if err != nil {
+		return nil, err
+	}
+
+	password, err := MongoPassword()
+	if err != nil {
+		return nil, err
+	}
+
+	timeoutSecStr := MongoConnectionTimeoutWithDefault(defaultMongoTimeoutSec)
+	timeoutSec, err := strconv.Atoi(timeoutSecStr)
+	if err != nil {
+		return nil, err
+	}
+
+	sslStr := MongoSSLWithDefault("true")
```

Comment:
can this instead of returning string, return a parsed bool and in case it is not parsable fallback to default provided one?